tests/unit/components/flash-message-test.js

```diff
@@ -26,10 +26,6 @@ moduleForComponent('flash-message', 'FlashMessageComponent', {
   },
 
   afterEach() {
-    run(() => {
-      flash.destroy();
```

Comment:
@poteto Let me know if this fix seems appropriate. It resolves all the failing tests in beta and alpha:
https://circleci.com/gh/poteto/ember-cli-flash/323?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link